src/cascadia/TerminalConnection/AzureConnection.idl

```diff
@@ -7,7 +7,9 @@ namespace Microsoft.Terminal.TerminalConnection
 {
     [default_interface] runtimeclass AzureConnection : ITerminalConnection
     {
-        AzureConnection(UInt32 rows, UInt32 columns);
+        static Boolean IsAzureConnectionAvailable();
+
+	AzureConnection(UInt32 rows, UInt32 columns);

```

Comment:
oh no this got worse somehow

Reply:
![github](https://user-images.githubusercontent.com/26824113/62327057-7427b780-b464-11e9-9717-1d07f75c41a1.png)

this is what it looks like on my computer... what is happening

Reply:
![image](https://user-images.githubusercontent.com/11050425/62327474-9110ba80-b465-11e9-928f-1aaab4c80bab.png)

Reply:
Tabs versus spaces? Visual Stu defaults to tabs for most file types.